kerckhoff/packages/models.py

```diff
@@ -33,13 +33,34 @@ class Meta:
         unique_together = ('package_set', 'slug',)
 
 # Snapshot of a Package instance at a particular time
+# A PackageVersion object is a specific combination of PackageItem objects
 class PackageVersion(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    package = models.ForeignKey(Package, on_delete=models.PROTECT)
-    creator = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    package = models.ForeignKey(Package, on_delete=models.CASCADE)
```

Comment:
whats the rationale here?

Reply:
Wouldn't we want to delete all package versions associated with a package if we delete that package?